docs/install_guide.adoc

```diff
@@ -0,0 +1,206 @@
+= User Installation Guide
+
+== Getting Started
+
+Before installing the Watcher operator you first need a functional
+OpenShift installation with the required Openstack operators,
+including the Telemetry operator. The following links point
+to documents detailing how to create this required starting environment:
+
+* https://github.com/openstack-k8s-operators/openstack-operator[Openstack Operator]
+* https://github.com/openstack-k8s-operators/telemetry-operator[Telemetry Operator]
+* https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operators]
+* https://prometheus.io/[Prometheus metrics]
+
+A CRC (https://crc.dev/docs/introducing/[Code Ready Containers]) installation is
+adequade for a developer environment. To verify that the environment set up is ready,
```

Comment:
I would suggest splitting this into a lead-in sentence followed by the first step. Something like:

To verify your environment, do the following:

.  Log in to the Kubernetes/Openshift environment:

(A period followed by a whitespace char indicates a numbered list in asciidoc.)